src/mapgen.h

```diff
@@ -114,6 +114,7 @@ enum MapgenType {
 	MAPGEN_V5,
 	MAPGEN_V6,
 	MAPGEN_V7,
+	MAPGEN_CARPATHIAN,
```

Comment:
if i remember this enum should be ordered by mapgen addition instead of names, @paramat can you confirm ?

It prevents mapgen ID mismatch